.golangci.yml

```diff
@@ -69,7 +69,7 @@ linters-settings:
       - name: unconditional-recursion
       - name: unexported-naming
       - name: unhandled-error
-        arguments: ["outputBuffer.Write", "fmt.Printf", "fmt.Println", "fmt.Print"]
+        arguments: ["outputBuffer.Write", "fmt.Printf", "fmt.Println", "fmt.Print", "rand.Read"]
```

Comment:
Can you explain this change?

Reply:
@powersj Sure.

We can read in documentation for `rand.Read` in `math/rand/rand.go` that `It always returns len(p) and a nil error.`
So I could add `//nolint` to such call or add this to exceptions (which I did).

But now I see that there are another calls to `rand.Read` in code. But from `crypto/rand/rand.go` - and this one can return error...

I will change to `//nolint`